wgengine/router/router_linux_test.go

```diff
@@ -314,8 +316,15 @@ ip route add throw 192.168.0.0/24 table 52` + basic,
 		},
 	}
 
+	mon, err := monitor.New(logger.Discard)
+	if err != nil {
+		t.Fatal(err)
+	}
+	mon.Start()
+	defer mon.Close()
+
 	fake := NewFakeOS(t)
-	router, err := newUserspaceRouterAdvanced(t.Logf, "tailscale0", fake.netfilter4, fake.netfilter6, fake, true, true)
+	router, err := newUserspaceRouterAdvanced(t.Logf, "tailscale0", mon, fake.netfilter4, fake.netfilter6, fake, true, true)
```

Comment:
not going to cause test flakes when a linkchange comes through during the test?

Reply:
Turns out this code didn't even compile before. It called a function which didn't exist in the code.

But we have no GOOS for which WireGuard compiles today but we don't.

So this is a useless change that can't currently be exercised.

Reply:
Well, it's currently unused, so that'd be impressive.

And with how I plan to use it, no.